README

```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
```

Comment:
We are using two abbreviations: cneuromod (short) and Courtois NeuroMod (long). I suggest to stick to Courtois NeuroMod here.